app/pods/publications/publication/proofs/documents/controller.js

```diff
@@ -16,6 +16,16 @@ const REQUEST_STAGES = {
   FINAL: 'final',
 };
 
+// necessary for tracking pieces
+export class Model {
+  @tracked pieces;
+  @tracked decisions;
+
+  constructor(models) {
+    Object.assign(this, models);
+  }
+}
```

Comment:
Met `RSVP.hash` (zie [Ember Guide multiple models](https://guides.emberjs.com/release/routing/specifying-a-routes-model/#toc_multiple-models)) werkt dit waarschijnlijk ook niet?

De core van het probleem waarom je deze workaround nodig hebt, zit in het feit dat het model gewijzigd wordt (vb. voor sorting en met `pushObject`) in de controller wat een anti-pattern is. Er staat een story op de backlog (KAS-2760) om dit op te lossen, dus ik ga het in deze PR zo laten voorbij gaan.